packages/@aws-cdk/app-staging-synthesizer-alpha/lib/default-staging-stack.ts

```diff
@@ -86,6 +87,14 @@ export interface DefaultStagingStackOptions {
    * @default - up to 3 versions stored
    */
   readonly imageAssetVersionCount?: number;
+
+  /**
+   * Auto deletes objects in the staging S3 bucket and images in the
+   * staging ECR repositories.
+   *
+   * @default false
+   */
+  readonly autoDeleteStagingAssets?: boolean;
```

Comment:
Does this need to be an option? I would argue the whole point of this change is that this can be the default behavior, which is what everyone wants.